site/en/guide/data_performance.ipynb

```diff
@@ -713,12 +713,12 @@
         "Here is a summary of the best practices for designing performant TensorFlow\n",
         "input pipelines:\n",
         "\n",
-        "*   [Use the `prefetch` transformation](#Pipelining) to overlap the work of a producer and consumer\n",
-        "*   [Parallelize the data reading transformation](#Parallelizing-data-extraction) using the `interleave` transformation\n",
-        "*   [Parallelize the `map` transformation](#Parallelizing-data-transformation) by setting the `num_parallel_calls` argument\n",
-        "*   [Use the `cache` transformation](#Caching) to cache data in memory during the first epoch\n",
-        "*   [Vectorize user-defined functions](#Map-and-batch) passed in to the `map` transformation\n",
-        "*   [Reduce memory usage](#Reducing-memory-footprint) when applying the `interleave`, `prefetch`, and `shuffle` transformations"
+        "*   [Use the `prefetch` transformation](#prefetching) to overlap the work of a producer and consumer\n",
```

Comment:
Automated anchor links for headings are generated differently on different systems, I don't remember the formatting for colab vs. github vs. tensorflow.org. It's best to include an <a name=""> at the target location so that it unambiguously works in  all contexts.

Reply:
Hi @MarkDaoust ,I don't have idea about the target location. In this case, Can I mention full URL of the working link?

Reply:
Sorry, my last comment didn't render correctly. 

`#Reducing-memory-footprint` is trying to point to `### Reduce memory footprint` heading. 

Please add something like `<a name="reduce-memory"> at each heading you're linking to, and update the link to match (like `[link](#reduce-memory)`). This is a good idea because each website normalizes the titles to anchors differently.

Reply:
You mean like below changes for the line - [Use the `prefetch` transformation](https://www.tensorflow.org/guide/data_performance#Pipelining) to overlap the work of a producer and consumer

 Use the `[<a name="prefetch">](#prefetching)` transformation 

or 

Use the `[prefetch](https://www.tensorflow.org/guide/data_performance#prefetching)` transformation

Reply:
No. For  ` [Reduce memory usage](#Reducing-memory-footprint)` 

go down to the section you're trying to link to and add the `<a>` there:

```
### Reduce memory footprint

<a name="reduce_memory_footprint">
```

Reply:
@MarkDaoust, Sorry for the delay. I have updated the file as you suggested. Please verify it once if it is same as described. Thank you.